snakebids/project_template/{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/run.py

```diff
@@ -11,7 +12,10 @@ def get_parser():
 
 
 def main():
-    app = SnakeBidsApp(Path(__file__).resolve().parent.parent)  # to get repository root
+    app = SnakeBidsApp(
+        Path(__file__).resolve().parent.parent,  # to get repository root
+        plugins=[BidsValidator],
```

Comment:
The class needs to be instantiated
```suggestion
        plugins=[BidsValidator()],
```